tests/ui/symbol-names/basic.legacy.stderr

```diff
@@ -1,10 +1,10 @@
-error: symbol-name(_ZN5basic4main17had874e876c8b1028E)
+error: symbol-name(_ZN5basic4main17h144191e1523a280eE)
```

Comment:
What's going on here? Because the new tykind variant is not at the end of the variants some later variants have their ids shifted?

Reply:
Yeah, every time we add a new variant this happens. You touched this test last when you added pattern types.